app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt

```diff
@@ -471,13 +473,12 @@ private fun assertKeyboardVisibility(isExpectedToBeVisible: Boolean) =
     )
 
 private fun navigationToolbar() =
-    onView(allOf(withText("Search or enter address")))
+    onView(withId(R.id.toolbar_wrapper))
```

Comment:
Wondering how this change makes test more robust or it is the main change the one in line 318 you added above?

Reply:
In this case I wanted to use the resource id of the **parent** (`toolbar_wrapper`) and not the child (`toolbar`) with text "Search or enter address"

I find using **resource id's** or **resource strings**  more reliable than using content description or text to query the desired items.